sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/MapKeys.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.transforms;
+
+import org.apache.beam.sdk.values.KV;
+import org.apache.beam.sdk.values.PCollection;
+
+/**
+ * {@code MapKeys} maps a {@code SerializableFunction} over keys of a {@code PCollection} of {@code
+ * KV<K1, V>}s and returns a {@code PCollection} of {@code KV<K2, V>}s.
```

Comment:
nit: I'd just make this docstring embrace the generics syntax
```suggestion
 * {@code MapKeys} maps a {@code SerializableFunction<K1,K2>} over keys of a {@code PCollection<KV<K1,V>>} and returns a {@code PCollection<KV<K2, V>>}s.
```